react/BaseContent.tsx

```diff
@@ -30,7 +30,7 @@ const Content: FC<Props> = ({ finishShoppingButtonLink }) => {
   const { navigate } = useRuntime()
 
   const goToCheckout = (url: string) => {
-    if (major) {
+    if (major && url === checkoutUrl) {
```

Comment:
nit: I would make it explicit that we want `major > 0` instead of just a truthy value.